.github/workflows/ci.yml

```diff
@@ -10,13 +10,19 @@ on:
       core-dump:
         description: "Set to 1 to enable retrieving core dump from crashes"
         default: "0"
+      gvsbuild-tag:
+        description: "Use an alternative gvsbuild release for the windows build. Defaults to latest."
+        default: "latest"
 jobs:
   test-linux:
-    runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python-version: ["3.7", "3.10"]
-
+        python-version: ["3.9", "3.10"]
+        os: ["ubuntu-24.04"]
+        include:
+          - os: ubuntu-22.04
+            python-version: 3.7
+    runs-on: ${{ matrix.os }}
```

Comment:
Let's keep these Linux CI changes separate from the Windows GTK changes

Reply:
That'll just prolong this whole thing. The linux changes are just: keep 22.04 (which needs an older python) and add a build on a newer version of ubuntu.

Reply:
Can we keep the scope of this PR to Win GTK only please